inbox/contacts/icloud.py

```diff
@@ -13,7 +13,7 @@
 from inbox.models.backends.generic import GenericAccount
 
 
-ICLOUD_CONTACTS_URL = 'https://contacts.icloud.com/'
+ICLOUD_CONTACTS_URL = 'https://contacts.icloud.com'
```

Comment:
Yeah - the problem is that we're just concatenating strings to build URLs, and at some point I assume the server started returning a URL fragment with the leading `/` on it. We could also fix it that way, but I've heard we may want to just delete all this code?